lib/bump/tasks.rb

```diff
@@ -14,8 +14,8 @@
       desc "Bump #{bump} part of gem version"
     end
 
-    task bump do
-      run_bump.call(bump, {})
+    task bump, :tag do |task, args|
```

Comment:
Unused block argument - `task`. If it's necessary, use `_` or `_task` as an argument name to indicate that it won't be used.